src/chunk.c

```diff
@@ -3549,6 +3547,17 @@ ts_chunk_unset_frozen(Chunk *chunk)
 #endif
 }
 
+bool
+ts_chunk_is_frozen(Chunk *chunk)
```

Comment:
Is this exported function intended for later use? I could not find any calls at the moment.

Reply:
called from the tsl code.